utils/actions/getOpenAISummary.ts

```diff
@@ -64,18 +64,28 @@ export default async function getOpenAISummary({
 
   const prompt = `Summarize what the ${summaryPrompt} ${
     commitList?.length ? `the ${commitList?.length} commits,` : ""
-  } are about. What do they tell us about the business logic? Don't summarize each piece or block of data separately, combine them and process all data. Take into consideration the current PR title and body. Don't look at each part or service of the list as a separate thing, but as a whole. The list will be available to me so you don't need to repeat it. Try to keep the summary to 3 or 4 sentences, but if it's a smaller thing just mention that.
-  Here is the list:\n  ${promptList} \n`;
+  } are about. What do they tell us about the business logic? Don't summarize each piece 
```

Comment:
Ah, maybe merge the commits into the prompt list?

Reply:
Sounds right to me. 

Are you talking about something like `promptList += commitList`

Reply:
Yes, as to have a standard way